recipes-extended/procps/procps_3.%.bbappend

```diff
@@ -0,0 +1 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Comment:
Procps was present in sumo but was probably not getting pulled in. Do we know what pulls it in now and if we really need the package?

Reply:
No I don't know what pulls it in and didn't spend much time trying to figure it out since its one of the packages that I tend to like to have installed anyway.  Does Toaster reveal that info ?

Reply:
Seems like kernel-dev is pulling in some dev packages that eventually pull in procps. Probably not worth figuring out what changed from sumo.

Do you think we should add an explicit dependency on procps from packagegroup-ni-base.bb so that pulling sysctl.conf is not left to chance in case something changes in future?

Reply:
That's probably not a bad idea at all, I'll add another commit to this PR.